src/client/reduxStore/sessionSaga.js

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+import * as types from './actionTypes'
+import {call, takeEvery} from 'redux-saga/effects';
+import { getAllMoviesAction } from './actions';
+
+function getMoviesFromInet() {
+    axios.get('https://reactjs-cdp.herokuapp.com/movies').then((data) => {
+      debugger;
```

Comment:
let's remove debugger. 
Actually you could use here async await and Fetch. You don't need here external library in order to do it